src/wp-includes/media.php

```diff
@@ -1059,24 +1059,37 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 		 */
 		$context = apply_filters( 'wp_get_attachment_image_context', 'wp_get_attachment_image' );
 
-		// Add `loading` attribute.
-		if ( wp_lazy_loading_enabled( 'img', $context ) ) {
-			$default_attr['loading'] = wp_get_loading_attr_default( $context );
-		}
-
 		$attr = wp_parse_args( $attr, $default_attr );
 
+		$loading_optimization_attr = wp_get_loading_optimization_attributes(
+			'img',
+			// $attr doesn't contain width and height by default.
+			array(
+				'width'  => $width,
+				'height' => $height,
+			) + $attr,
```

Comment:
```suggestion
                 $loading_attr = $attr;
                 $loading_attr['width']  = $width;
                 $loading_attr['height']  = $height;
		$loading_optimization_attr = wp_get_loading_optimization_attributes(
			'img',
			$loading_attr,
```
This is a little more readable.